features/exception_handling/developer_is_told_which_scenario_fataled.feature

```diff
@@ -0,0 +1,45 @@
+@isolated
+Feature: As a Developer
```

Comment:
``As a developer`` should not be the feature title but the first line of the description (and the feature needs a proper title)